### PR TITLE
fix fail to read settings from settings file

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- title
- bug is old, but only circumstances reveal it in 3.11.0 with CBA only
- Thanks to cyruz, diwako, max255pl and whoever I forgot for helping with debugging and thanks for reporting ofc :P

### CODE REVIEW

**Comment by diwako** [comment]
Tested on dedicated linux server. Server enforced settings for ACE and ACRE2 during the test as expected from the cba_settings.sqf file in a PBO loaded on server.

File: addons/settings/fnc_addSetting.sqf [hunk 1]
```diff
@@ -45,4 +45,8 @@ Author:
     commy2
 ---------------------------------------------------------------------------- */
 
-call (uiNamespace getVariable QFUNC(init)) == 0
+if (isNil QFUNC(init)) then {
```
Comment (jonpas) [inline addons/settings/fnc_addSetting.sqf]: Comment why?

Comment (commy2) [reply addons/settings/fnc_addSetting.sqf]: If this one function undefined (implies all others are too) then define all functions.
It's basically an include guard.

Comment (jonpas) [reply addons/settings/fnc_addSetting.sqf]: Not comment for me (but probably also for me in a few months).

**Pull request merged**